src/app/shell/personal-cabinet/provider/provider-admins/provider-admins.component.ts

```diff
@@ -51,45 +43,42 @@ export class ProviderAdminsComponent extends ProviderComponent implements OnInit
   providerAdmins$: Observable<SearchResponse<ProviderAdmin[]>>;
   providerAdmins: SearchResponse<ProviderAdmin[]>;
   providerAdminsData: ProviderAdminTable[] = [];
-  @Select(PaginatorState.tableItemsPerPage)
-  tableItemsPerPage$: Observable<number>;
 
   filterFormControl: FormControl = new FormControl('');
   currentPage: PaginationElement = PaginationConstants.firstPage;
   tabIndex: number;
 
-  private filterParams: ProviderAdminParameters = {
+  filterParams: ProviderAdminParameters = {
     assistantsOnly: false,
     deputyOnly: false,
     searchString: '',
+    size: PaginationConstants.TABLE_ITEMS_PER_PAGE
   };
 
-  constructor(
-    protected store: Store,
-    protected matDialog: MatDialog,
-    private router: Router,
-    private route: ActivatedRoute
-  ) {
+  constructor(protected store: Store, protected matDialog: MatDialog, private router: Router, private route: ActivatedRoute) {
     super(store, matDialog);
   }
 
   ngOnInit(): void {
     super.ngOnInit();
+    Util.setFromPaginationParam(this.filterParams, this.currentPage);
+
     this.route.queryParams.pipe(takeUntil(this.destroy$), debounceTime(500)).subscribe((params: Params) => {
       this.tabIndex = params['role'] ? Object.keys(this.providerAdminRole).indexOf(params['role']) : 0;
       this.filterParams.assistantsOnly = params['role'] === ProviderAdminRole.admin;
       this.filterParams.deputyOnly = params['role'] === ProviderAdminRole.deputy;
```

Comment:
I belive here you should set the first page, correct?
because entering new text will give you no result on the second page